### ISSUE TITLE
routerconfigs not running daily schedule for some devices since upgrading Cacti to 1.2.27

### ISSUE THREAD

**Issue by jdcoats** [Cacti/plugin_routerconfigs#131]
I upgraded Cacti to 1.2.27 on 5/7/2024 Since then I'm not getting the daily run emails. When I check the devices i see about half of them the next run date hasn't changed. Manual run works but still does not update the next run date.
![image](https://github.com/Cacti/plugin_routerconfigs/assets/42470924/cde0c91f-78d3-4cb2-b87c-81a294a70336)
The unaffected devices still backup on schedule but i no longer receive the report email.
![image](https://github.com/Cacti/plugin_routerconfigs/assets/42470924/8fd4b1dc-d68f-4989-bba8-29fd191a9de7)

**Comment by jdcoats** [comment]
I manually updated the next run date for all of the affected devices like
```
UPDATE `plugin_routerconfigs_devices` SET `nextbackup` = '1715904000' WHERE `plugin_routerconfigs_devices`.`id` =245;
UPDATE `plugin_routerconfigs_devices` SET `nextbackup` = '1715904000' WHERE `plugin_routerconfigs_devices`.`id` =246;
UPDATE `plugin_routerconfigs_devices` SET `nextbackup` = '1715904000' WHERE `plugin_routerconfigs_devices`.`id` =247;
UPDATE `plugin_routerconfigs_devices` SET `nextbackup` = '1715904000' WHERE `plugin_routerconfigs_devices`.`id` =248;
```
and they still didn't backup yesterday.
![image](https://github.com/Cacti/plugin_routerconfigs/assets/42470924/893eecd4-da76-43cb-819d-170d59fea936)

**Comment by jdcoats** [comment]
okay I figured this one out and it happens that on the same day I upgraded one of my voice gateways got in a stuck state where it was functional in all aspects but ssh. The ssh session would connect but just sit there never returning a prompt indefinitely. This would hang the routerconfigs at that device id and never complete. I had to kill all of the hung processes and disable that faulty device to get my scheduled runs to complete.
